packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1009,9 +1010,55 @@ export const standardLinterfunctions: FunctionItem[] = [
     functionName: 'apilintOpenAPIPathTemplateValid',
     function: (element: Element) => {
       if (isStringElement(element)) {
-        return true;
+        const pathItemElement = (element.parent as MemberElement).value as ObjectElement;
+
+        if (pathItemElement.length === 0) {
+          return true;
+        }
+
+        const parameterElements: Element[] = [];
+        const isParameterElement = (el: Element): boolean => el.element === 'parameter';
+
+        pathItemElement.forEach((el) => {
+          if (el.element === 'parameters') {
```

Comment:
Just found this **bug**. This will never be true. You can try creating a failing test for this. Basically there is no ApiDOM element with `parameters` type. `PathItem.parameters` is an `ArrayElement` with `path-item-paramters` meta class. What you're interested here is asserting on `PathItemParametersElement` NCE (Non Concrete Element type).

NCEs:
 - OpenAPI 2.0 - https://github.com/swagger-api/apidom/blob/main/packages/apidom-ns-openapi-2/src/elements/nces/PathItemParameters.ts
 - OpenAPI 3.x.y - https://github.com/swagger-api/apidom/blob/main/packages/apidom-ns-openapi-3-0/src/elements/nces/PathItemParameters.ts

```suggestion
          if (includesClasses(['path-item-parameters'], el)) {
```

---

`includesClasses` is export from `@swagger-api/apidom-core` package.

Reply:
But at this point IMHO it make more sense to pull the first `if` statement outside of the loop and just process it manually as I did in the "semantic code template". You will avoid asserting for meta class on every `Path Item` fixed field and converting meta classes to POJOS (done by `includesClasses`).

Reply:
Right. I initially did it that way but then I put the check inside loop as I thought it looked cleaner and I wasn't aware of the implications of it. 
Changed.